### PR TITLE
Switched from Connection to WebservicesAuthentication

### PR DESCRIPTION
**Pull request by Waterdhavian**
They changed the way login works again

### CODE REVIEW

**Comment by Waterdhavian** [comment]
I only made credentials work, if you want to use oauth then the logic needs to be expanded and include the oauth authentication provider as well.

File: src/ApiConnectorFactory.php [hunk 1]
```diff
@@ -63,43 +62,13 @@ public function get($name): BaseApiConnector
     }
 
     /**
-     * Get the Twinfield Connection.
+     * Get the Twinfield AuthenticatedConnection.
      *
-     * @return Connection
+     * @return AuthenticatedConnection
```
Comment (willemo) [inline src/ApiConnectorFactory.php]: This seems not to be a valid class name

**Review by willemo — CHANGES_REQUESTED** [review]
I've found one possible issue in this PR

**Comment by Waterdhavian** [comment]
You are correct, my bad. Fixed.

**Comment by willemo** [comment]
Thanks for this PR!

**Comment by Waterdhavian** [comment]
You're welcome!

**Pull request merged**